dotnet-desktop-guide/framework/wpf/advanced/how-to-override-the-logical-tree.md

```diff
@@ -5,6 +5,7 @@ helpviewer_keywords:
   - "overriding the logical tree [WPF]"
   - "logical tree [WPF], overriding"
 ms.assetid: 0ae4d074-8113-4b06-b4fa-e0f39d4967a6
+description: Learn how to subclass StackPanel to override the logical tree and enforce a behavior that the panel may only have and will only render a single child element.
```

Comment:
Add a comma based on the correct grouping... I believe after the word "have": 
Learn how to subclass StackPanel to override the logical tree and enforce a behavior that the panel may only have, and will only render a single child element.